frontend/js/src/about/add-data/AddData.tsx

```diff
@@ -367,11 +375,15 @@ export default function AddData() {
         into automatic Spotify submissions you may notice duplicates for your
         last 50 Spotify listens.
       </p>
-      <p>
-        <Link to="/settings/music-services/details/">
-          Connect your Spotify or Last.fm account to ListenBrainz
-        </Link>
-      </p>
+    <ul>
+        <li>
+          <em>
+            <a href="/settings/music-services/details/">
+              Connect your Spotify or Last.fm account to ListenBrainz
+            </a>
```

Comment:
We want this link to stay the same type of element it was before, it is for internal navigation within the single-page app.
```suggestion
            <Link to="/settings/music-services/details/">
              Connect your Spotify or Last.fm account to ListenBrainz
            </Link>
```